bin/date

```diff
@@ -103,7 +103,7 @@ sub select_format {
 
 sub setup_specifiers {
 	my %specifiers = (
-		'e'  => sprintf( '% 2d', (core_time)[3] ),
```

Comment:
Huh, I did not know the space acted like that, but looking at the docs, that's what it does.